src/content/app/tools/blast/views/blast-submission-results/components/single-blast-job-result/SingleBlastJobResult.scss

```diff
@@ -28,6 +28,30 @@
   }
 }
 
+.failedJobSummaryRow {
+  display: grid;
+  grid-template-columns:
+    [empty] 11ch [failed-status] minmax(48%, 1050px)
+    [species] minmax(20%, 1fr);
+
+  grid-template-areas: 'empty failed-status species';
+
+  gap: 20px;
+  padding-bottom: 40px;
+
+  &:first-of-type {
+    padding-top: 24px;
+  }
+
+  &:last-of-type {
+    padding-bottom: 46px;
+  }
+
+  .blastSpecies {
+    color: $dark-grey;
+  }
+}
```

Comment:
Wondering if we can just add an extra class to the corresponding element (`.resultsSummaryRow.failedJobRow`). so that all the padding and stuff comes from `.resultsSummaryRow` (rather than being copy-pasted), and only the grid settings are updated by a `failedJobRow` class

Reply:
lets discuss this together with the above css suggestion.